kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
I wonder if we can set commit hash string instead of `-SNAPSHOT` in `pom.xml` file (e.g. 2.3.0-f9d44a838b26f537ea65fd36b5eeb6d07fb5d39f). It is allowed in semver. The script that suppose to build the container can use that specific commit or a released version otherwise.